src/main/java/nvnieuwk/nf/test/bam/Methods.java

```diff
@@ -44,6 +52,55 @@ public static AlignmentFile bam(LinkedHashMap<String,Object> options, CharSequen
 			} else {
 				createRefIndex(referencePath);
 			}
+
+		} else if(refString.startsWith("s3://") || refString.startsWith("s3:/")) {
```

Comment:
```suggestion
		} else if(refString.startsWith("s3:/")) {
```

The first one is a bit redundant :)